src/pyhf/exceptions/__init__.py

```diff
@@ -68,6 +68,12 @@ class ImportBackendError(Exception):
     """
 
 
+class InvalidBackend(Exception):
+    """
+    InvalidBackend is raised when trying to set using a backend that does not exist.
```

Comment:
"trying to set using"

Reply:
? Sorry I'm not sure what your comment is suggesting.

Reply:
this isn't a grammatically correct sentence, and it reads awkwardly.

Reply:
I copied the [docstring you wrote for `InvalidOptimizer`](https://github.com/scikit-hep/pyhf/pull/403/files#diff-37200bfdeb3cc4323cb5f51ba7cd9670R77) and then just changed "optimizer" to "backend". Do you want to change both at the same time?

How about
```
InvalidX is raised when trying to set a X that does not exist.
```

?

Reply:
that sounds better